frontend/src/app/api/questions/[id]/route.ts

```
@@ -0,0 +1,131 @@
+    //   return NextResponse.json(
+    //     { error: "Unauthorized" },
+    //     { status: 401 }
+    //   );
+    // }
+
+    const xata = getXataClient();
+    
+    // Fetch the question with its area relationship
+    const xata = getXataClient();
+    const questionId = parseInt(params.id)
+    if (isNaN(questionId) || questionId <= 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)